### PR TITLE
Update quantum workspace creation instructions for "Credits for All" providers

### PR DESCRIPTION
**Pull request by warren-jones**

### CODE REVIEW

**Comment by PRMerger4** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Court72** [comment]
@ricardo-espinoza 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by PRMerger19** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md [hunk 1]
```diff
@@ -59,16 +59,28 @@ To create a new Azure Quantum workspace, you'll need to know:
...
 1. To retrieve the list of available quantum providers, use the `list` command (this example uses **westus** as the location):
 
    ```azurecli
    az quantum offerings list -l westus -o table
    ```
 
+> [!TIP]
+> If want to see which providers give free credit, use the `--autoadd-only` parameter, for example:<br />
+> `az quantum offerings list --autoadd-only -l westus -o table`<br />
+> As mentioned above, those providers are automatically added to your workspace. You do not need to specify them with the `-r` parameter.
```
Comment (Bradben) [inline articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:82]:
```suggestion
        > [!TIP]
        > If want to see which providers give free credit, use the `--autoadd-only` parameter, for example:<br />
        > `az quantum offerings list --autoadd-only -l westus -o table`<br />
        > As mentioned previously, those providers are automatically added to your workspace. You do not need to specify them with the `-r` parameter.
```

**Review by Bradben — APPROVED** [review]
Minor edits. Thanks!

**Comment by PRMerger6** [comment]
@warren-jones : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Bradben — APPROVED** [review]

File: articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md [hunk 2]
```diff
@@ -59,16 +59,28 @@ To create a new Azure Quantum workspace, you'll need to know:
 - The resource group associated with the new workspace, for example, **MyResourceGroup**.
 - A storage account in the same resource group and subscription as the quantum workspace. It's possible to [create a new storage account from the Az CLI tool](/cli/azure/storage/account#az_storage_account_create), for example, **MyStorageAccount**.
 - The name of the quantum workspace to create, for example, **MyQuantumWorkspace**.
-- The list of Azure Quantum providers to use in the workspace. A provider offers a set of plans, each of them representing a plan with associated terms and conditions, costs, and quotas. To create workspaces, you'll need to specify the corresponding plan along with the providers.
+- The list of Azure Quantum providers to use in the workspace. A provider offers a set of plans, each of them representing a plan with associated terms and conditions, costs, and quotas. To create workspaces, you'll need to specify the corresponding plan along with the providers, unless you want to start with the providers that offer free credit – they are automatically added to your workspace.
```
Comment (shenyingjun) [inline articles/how-to-manage-quantum-workspaces-with-the-azure-cli.md:62]: i prefer state that providers that offer free credit is the default option to encourage user leverage this feature.

**Review by shenyingjun — APPROVED** [review]

**Pull request merged**